Mathlib/MeasureTheory/Measure/MeasureSpaceDef.lean

```diff
@@ -370,15 +375,22 @@ def Measure.ae {α : Type*} {_m : MeasurableSpace α} (μ : Measure α) : Filter
 #align measure_theory.measure.ae MeasureTheory.Measure.ae
 
 -- mathport name: «expr∀ᵐ ∂ , »
+/-- `∀ᵐ a ∂μ, p a` means that `p a` for a.e. `a`, i.e. `p` holds true away from a null set -/
```

Comment:
```suggestion
/-- `∀ᵐ a ∂μ, p a` means that `p a` for a.e. `a`, i.e. `p` holds true away from a null set.

 This is notation for `Filter.Eventually p (Measure.ae μ)`. -/
```
since we can't access the definition from the hover. Same below